src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -85,7 +85,11 @@ public record ProxyHeartbeat
     Guid ProxyId,
     List<Forward> Forwards,
     DateTimeOffset TimeStamp
-);
+) {
+    public override string ToString() {
+        return JsonSerializer.Serialize(this);
+    }
```

Comment:
`ProxyHeartbeat.ToString()` was being used in some log messages but the `List<Forwards>` was being serialized as ```System.Collections.Generic.List`1[Microsoft.OneFuzz.Service.Forward]```